vendor/bat-native-ads/src/bat/ads/internal/legacy_migration/database/database_migration.cc

```diff
@@ -69,6 +70,9 @@ void Migration::ToVersion(mojom::DBTransaction* transaction,
   table::AdEvents ad_events_database_table;
   ad_events_database_table.Migrate(transaction, to_version);
 
+  table::TextEmbeddingHTMLEvents text_embedding_html_events_database_table;
```

Comment:
nit: `TextEmbeddingHTMLEvents` => `TextEmbeddingHtmlEvents`

Reply:
For this one I tried to keep consistency with `AdsImpl::OnHtmlLoaded`

Reply:
Updated where relevant